From 885f6eab467695d2bf116cc356adfb82d50f38d2 Mon Sep 17 00:00:00 2001
Message-Id: <885f6eab467695d2bf116cc356adfb82d50f38d2.1415245453.git.chang-joon.lee@intel.com>
In-Reply-To: <76203ad7c635d495ef3ec1e07b47bad9db4745ba.1415245452.git.chang-joon.lee@intel.com>
References: <76203ad7c635d495ef3ec1e07b47bad9db4745ba.1415245452.git.chang-joon.lee@intel.com>
From: Michel Thierry <michel.thierry@intel.com>
Date: Mon, 3 Nov 2014 12:28:53 +0000
Subject: [PATCH 27/30] MUST_REBASE [VPG]: drm/i915/vlv: Do not use PPGTT in
 vlv

At least aliasing PPGTT has been enabled for Gen7 (VLV) upstream.
We don't want/need this for GMIN-L as it was never part of the plan of
record.

Therefore we need a must rebase patch to disable PPGTT again.

Issue: GMINL-3506
Change-Id: If6eab38012d8a8940d58e05e1041d2ef42ac2e39
Signed-off-by: Michel Thierry <michel.thierry@intel.com>
---
 drivers/gpu/drm/i915/i915_gem_gtt.c |    7 +++----
 1 file changed, 3 insertions(+), 4 deletions(-)

diff --git a/drivers/gpu/drm/i915/i915_gem_gtt.c b/drivers/gpu/drm/i915/i915_gem_gtt.c
index fce58a7..2ed9d01 100644
--- a/drivers/gpu/drm/i915/i915_gem_gtt.c
+++ b/drivers/gpu/drm/i915/i915_gem_gtt.c
@@ -60,10 +60,9 @@ static int sanitize_enable_ppgtt(struct drm_device *dev, int enable_ppgtt)
 	}
 #endif
 
-	/* Early VLV doesn't have this */
-	if (IS_VALLEYVIEW(dev) && !IS_CHERRYVIEW(dev) &&
-	    dev->pdev->revision < 0xb) {
-		DRM_DEBUG_DRIVER("disabling PPGTT on pre-B3 step VLV\n");
+	/* VLV Android doesn't want this */
+	if (IS_VALLEYVIEW(dev) && !IS_CHERRYVIEW(dev)) {
+		DRM_DEBUG_DRIVER("disabling PPGTT on VLV\n");
 		return 0;
 	}
 
-- 
1.7.9.5

